conans/client/rest/uploader_downloader.py

```diff
@@ -135,6 +136,8 @@ def download(self, url, file_path=None, auth=None, retry=1, retry_wait=0, overwr
         if not response.ok:  # Do not retry if not found or whatever controlled error
             if response.status_code == 404:
                 raise NotFoundException("Not found: %s" % url)
+            elif response.status_code == 401:
```

Comment:
It wasn't a problem with apiv1 because downloading the manifest was a special `/digest` api call. Now is a generic file download (because we can do it being decoupled). So, it was a bug but not affecting.